COMMITTERS.adoc

```diff
@@ -0,0 +1,182 @@
+  and see how things evolve.
+  This should give you some hints of important things to check in the future.
+* Only merge pull requests that target master.
+  **Dealing with old branches is a separate process.**
+* If you are sure a PR should go in and is just waiting for CI,
+  just use the `triage/waiting-for-ci` label.
+  It is a good hint for other committers.
+* If you think a pull request is an essential part of a given milestone
+  (and can realistically be part of it),
+  you should affect this milestone to it so that we can be sure it stays on the radar.
+
+Obviously, each situation is different so use your own judgement,
+and if in doubt, just ask for advice, the other committers are here to help.
+
+== Pull Request Title
+
+The title should be in English, should not contain an issue number.
+It should also not contain ellipsis.
+
+If your commit message was too long and GitHub automatically cut
+the title, that's helpful if you can take the time to move the cut
+part where it belongs to have a full title.
+
+Titles are included in the Release notes so they are important.
+
+A good title would look like: `Fix XXX in Quartz extension` or
+`Introduce Hibernate Reactive extension`.
+
+A bad title would look like: `fix(#444)`.
+
+== Issues Fixed
+
+When a PR fixes some issues, it's good practice to add it in the description.
+
+One issue per line with something like:
+
+[source,asciidoc]
+----
+* Fix #444
+* Fix #555
+----
+
+[TIP]
+====
+GitHub supports a variety of keywords here: `fix`, `fixes`, `fixed`,
+`resolve`, `resolves`, `resolved`, `close`, `closes`, `closed`
+all do the same thing.
+====
+
+[WARNING]
+====
+GitHub won't detect issues properly if you do something like
+`Fix #444 #555`.
+====
+
+== Affecting Labels and Milestones
+
+Affecting labels and milestones is very important in our process.
+
+Before each release, we check that we don't have any orphan i.e.
+closed issues that don't have either a milestone affected or
+some of the "excluding" labels.
+
+Thus:
+
+* When you merge a pull request, please affect it with the current milestone marked with "master".
+* If the pull request has issues associated (i.e. if the pull request fixes some issues),
+  you should also affect the milestone assigned to the issues.
+* If you close a pull request because the committers have decided to not merge it,
+  please add the appropriate `triage/` label: `triage/invalid`, `triage/out-of-date`,
+  `triage/wontfix` are usually in order.
+* If you close an issue because it has been fixed, either add the milestone
+  if you can find it easily or use the `triage/out-of-date` if you can't.
+* If you close an issue for any other reason, one of the aforementioned `triage/` labels
+  is probably adequate.
+
+== Backporting Process
+
+When we release a new version of Quarkus, we usually do a bugfix
+release a couple of weeks after.
+
+Everytime we do a release, we create a release branch to host
```

Comment:
we should mention the naming pattern used.

Reply:
Well, it's the version. What's the point in adding that info here? (The document is already very long)

Reply:
```suggestion
Everytime we do release (i.e. 1.6.0), we create a branch (i.e. 1.6) to host
```

I don't think that piece of clarity and ~10 characters hurts ;) up to you.

Reply:
@maxandersen There is an extra `do` in your suggestion (eg. Everytime we release...) :smiley:

Reply:
It's not the extra characters, it's just that I don't think it brings anything to the plate and just adds noise. Did it anyway in the last version.